src/recommendations/product-recs.md

```diff
@@ -0,0 +1,53 @@
+---
+group: product-recommendations
+title: Product Recommendations
+---
+
+Product recommendations are a powerful marketing tool you can use to increase conversions, boost revenue, and stimulate shopper engagement. Recommendations are surfaced on the storefront in the form of units such as “Customers who viewed this product also viewed”. Because these suggestions are backed by a deep analysis of aggregated visitor data, they result in highly engaging, relevant, and personalized experiences for the shopper. Magento does not collect personally identifiable information.
+
+Using recommendations on your storefront requires:
```

Comment:
I would like to see this list be more active. Adobe Sensei does X, Y, and Z to compute product affinites. Magento combines static product metadata with Sensei data to do something.

And omit the Admin UI from the list.